src/pages/home/components/Sidebar/components/SocialAccordion/SocialAccordion.type.ts

```diff
@@ -1,8 +1,12 @@
+import { ReactElement } from 'react';
+
 import { StoreAccount } from '~stores/useSocialMediaStore/useSocialMediaStore.types';
 
 export type SocialAccordionProps = {
   accounts: StoreAccount[];
   error: boolean;
+  icon?: ReactElement;
+  isFavoriteAccordion?: boolean;
```

Comment:
Não gosto dessa prop `isFavoriteAccordion` tira versatilidade e escalabilidade do componente, se formos ter outros formatos vamos colocar esses outros `is` tambem? A solução aqui é ter uma `prop` de `name` ou `title` pq ai o componente, independente de quantas variações ele tenha, sempre vai funcionar com pouca ou até (provavelmente) nenhuma alteração do componente em si.